clients/rospy/src/rospy/impl/transport.py

```diff
@@ -93,15 +93,16 @@ def fileno(self):
     ## closed. Function will be passed this transport as an argument.
     ## @param self
     ## @param cleanup_callback fn(Transport): callback for when connection is closed
-    def set_cleanup_callback(self, cleanup_callback):
-        self.cleanup_cb = cleanup_callback
+    def add_cleanup_callback(self, cleanup_callback):
```

Comment:
This is a breaking API change which can not be merged into already released distro. Maybe the new function `add_cleanup_callback` should be added and the old function can call the new function to keep it kind of compatible?